tests/model/test_model.py

```diff
@@ -652,7 +652,7 @@ def test__stream_info_from_subscriptions(self, initial_data, streams,
 
     def test__handle_message_event_with_Falsey_log(self, mocker,
                                                    model, message_fixture):
-        model.found_newest = True
+        model.set_found_newest(True)
```

Comment:
Note my other comment about whether we need this function, but it may be worth investigating clarifying these tests, perhaps adding notes why these are set, or changing the test names to be clearer.